device/common/profiles/th5/gen/BALANCED/buffers_defaults_t1.j2

```diff
@@ -5,32 +5,37 @@
 {%- macro generate_buffer_pool_and_profiles() %}
     "BUFFER_POOL": {
         "ingress_lossless_pool": {
+            "size": "60000000",
             "type": "ingress",
             "mode": "dynamic",
-            "size": "60000000",
             "xoff": "22600000"
         },
+        "egress_lossy_pool": {
+            "size": "41300000",
+            "type": "egress",
+            "mode": "dynamic"
+        },
         "egress_lossless_pool": {
+            "size": "41300000",
```

Comment:
in the sai definition, we do not have ITM concept. we treat whole device as pool, and therefore i am not sure we need to cut this in half is satisfying the sai definition.

Reply:
I think you're right, but these are just placeholder values to fix a SAI crash anyways. We'll be updating these with tuned values shortly.